hw/xbox/smbus_xbox_smc.c

```diff
@@ -181,14 +181,34 @@ static uint8_t smc_receive_byte(SMBusDevice *dev)
 static void smbus_smc_realize(DeviceState *dev, Error **errp)
 {
     SMBusSMCDevice *smc = XBOX_SMC(dev);
+    char *avpack;
 
     smc->version_string_index = 0;
+    /* pretend to have a composite av pack plugged in by default */
+    smc->avpack_reg = SMC_REG_AVPACK_COMPOSITE;
     smc->scratch_reg = 0;
     smc->cmd = 0;
 
     if (object_property_get_bool(qdev_get_machine(), "short-animation", NULL)) {
         smc->scratch_reg = SMC_REG_SCRATCH_SHORT_ANIMATION;
     }
+
+    avpack = object_property_get_str(qdev_get_machine(), "avpack", NULL);
+    if (strcmp(avpack, "scart") == 0) {
+        smc->avpack_reg = SMC_REG_AVPACK_SCART;
+    }
+    else if (strcmp(avpack, "svideo") == 0) {
+        smc->avpack_reg = SMC_REG_AVPACK_SVIDEO;
+    }
+    else if (strcmp(avpack, "vga") == 0) {
+        smc->avpack_reg = SMC_REG_AVPACK_VGA;
+    }
+    else if (strcmp(avpack, "vga-sog") == 0) {
+        smc->avpack_reg = SMC_REG_AVPACK_VGA_SOG;
```

Comment:
What does SOG stand for?

Reply:
Sync on green

Reply:
> https://xboxdevwiki.net/Video_Encoder

Sync-on-Green maybe.

Reply:
These constants are weird, see https://xboxdevwiki.net/AV_Cables for the MS names in the respective AV code + more notes.